webpack.config.js

```diff
@@ -9,7 +9,11 @@ module.exports = webpackConfigEnv => {
     webpackConfigEnv
   });
 
-  return webpackMerge.smart(defaultConfig, {
+  const rxjsExternals = {
```

Comment:
Out of curiosity, why not just put this into the object on line 16 through 38

Reply:
TBH? No reason.

Reply:
I was playing around with how it worked and I did it this way just for fun. I can change it if you'd prefer it inline.